### ISSUE TITLE
Permissions not registered by LuckPerms

### ISSUE THREAD

**Issue by lucaszischka** [GeyserMC/Floodgate#150]
If you try to see all registered Permissions, using `/lp tree`, you can see that the permissions are not registered by luckperms; after the usage of a command, the associated permission appears. This causes problems, because at one it is more difficult to see and edit all floodgate and geyser permissions, but also everyone can see the floodgate and geyser commands (via tab-complete), even if he doesn't have permission to use them.

It would be really helpful if the commands could be registered.

The relevant plugin and server info:
```
System Info: Java 16 (OpenJDK 64-Bit Server VM 16.0.1+9) Host: Windows 10 10.0 (amd64)
[12:33:31 INFO]: This server is running Paper version git-Paper-741 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)
[12:33:36 INFO]: [LuckPerms] Loading LuckPerms v5.3.42
[12:33:36 INFO]: [floodgate] Loading floodgate v2.0-SNAPSHOT
[12:33:38 INFO]: [Geyser-Spigot] Loading Geyser-Spigot v1.3.0-SNAPSHOT
```

If you need more system and server info please don't hesitate to ask; also it may be helpful to add that everything works fine except the permission registration; no errors or problems whatsoever.

Regards Lucas

**Comment by DarkMagicSource** [comment]
I've been having a similar issue. I think it's universal for most/all permission systems having issues detecting the commands.

**Comment by Tim203** [comment]
This should be fixed in `dev/2.1.1`
